README.md

```diff
@@ -0,0 +1,19 @@
+# I AM
+SPIRITED
+BELIEVER
+FAITHFUL
+LOVING
+WRITER
+MUSICIAN
+SINGER
+EMPATH
+MEDIUM
+CHRISTIAN
```

Comment:
CHRISTIANITY IS TO SCIENECE WHAT SCIENCE IS TO CHRISTIANITY - THEY ARE ONE IN THE SAME.  GOD IS LIGHT.  LIGHT IS THE WAY.